virtcontainers/sandbox.go

```diff
@@ -1062,6 +1068,26 @@ func (s *Sandbox) stopVM() error {
 		s.Logger().Info("Not stopping VM")
 		return nil
 	}
+	pid := s.hypervisor.pid()
+
+	defer func() {
+		if err != nil {
+			networkLogger().WithError(err).WithField("pid", pid).Warn("Fail to stop vm")
+		}
+		for i := 0; i < waitRetry; i++ {
+			if running, _ := isProcRunning(pid); !running {
+				err = nil
+				return
+			}
+			syscall.Kill(pid, syscall.SIGKILL)
```

Comment:
SIGKILL is not catchable. Please only send it once.

Reply:
> Please only send it once.
 
@Ace-Tang what do you think? Have you seen cases where the first signal `SIGKILL` doesn't kill the hypervisor?

Reply:
I am not seen qemu not be killed by SIGKILL

Reply:
Updated, @devimc @bergwolf